src/components/viewSections/LicenseAmendments.js

```diff
@@ -22,12 +22,14 @@ export default class LicenseAmendments extends React.Component {
       onAmendmentClick: PropTypes.func,
     }),
     id: PropTypes.string,
+    licenseAmendmentsAccordionLabel: PropTypes.string,
```

Comment:
FormattedMessage returns us a React element. So this needs to be a type of element or if you want to allow strings as well in the future, we can do 

```suggestion
    licenseAmendmentsAccordionLabel: PropTypes. oneOfType([
        PropTypes.string,
        PropTypes.element,
    ]),
```